guides/micronaut-metrics-aws/src/test/resources/application-test.yml

```
@@ -0,0 +1,7 @@
+crypto:
+  initialDelay: 10h # <1>
```

```suggestion
  initialDelay: 10h
```
We don't show the file in the guide. We don't need a callout.